### PR TITLE
 feat: adds a section about Merging Conflicts

### LINKED ISSUE #81: Feature: add section for resolving merge conflicts
### Type of feature

🍕 Feature

### Current behavior

Under the [How to Contribute to Open Source](https://intro.opensauced.pizza/#/05-how-to-contribute-to-open-source?id=how-to-contribute-to-open-source) section, there is a lot of helpful information but it seems to miss a section on how to resolve merge conflicts.

### Suggested solution

I think this is an important section to include because it is one of the common issues I have seen as a maintainer. 
I think adding a couple of sentences on what merge conflicts are and how to resolve them inside the VS code editor would be helpful 👍 

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [x] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by Lymah123**
## This PR 
- This PR adds a new section about Merge conflicts to the Readme

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
Closes #81 

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [x] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by Lymah123** [comment]
@CBID2 and @BekahHW . I will be waiting for your reviews.

Thank you.

File: _layouts/sidebar.md
Comment (CBID2) [inline _layouts/sidebar.md:12]: This appeared when I clicked on the Conflicts section
D1B913CB-5A06-4CD5-B35C-D6C3673BA99C

**Comment by Lymah123** [comment]
> Hi @Lymah123. Just gave you a comment

Okay @CBID2 , I will look into that.

**Comment by Lymah123** [comment]
https://www.loom.com/share/171d34d2e18c48f4bb90b226770608a8

Hi @CBID2 , I am having issue with the markdown. Can you see what I am facing in the video clip?

**Comment by Lymah123** [comment]
> > https://www.loom.com/share/171d34d2e18c48f4bb90b226770608a8
> > Hi @CBID2 , I am having issue with the markdown. Can you see what I am facing in the video clip?
> 
> I think that's because you did not add a file name and end it with `.md`

Wow! it worked. I forgot totally! Thank you @CBID2 !

**Comment by Lymah123** [comment]
Hi @CBID2 , you can review again.

Thanks a lot.

**Comment by Lymah123** [comment]
> Hey @Lymah123, when you have a minute, can you take a look at the [docs for PRs ](https://docs.opensauced.pizza/contributing/introduction-to-contributing/#pull-requests)? There's some info there I think you'll find helpful on creating PRs.

Alright. Thank you @BekahHW !

**Review by CBID2 — APPROVED** [review]
Well done @Lymah123! :) I made some more edits to your PR's form. As you continue your open source journey, always fill out every section of the pull request form based on the commented instructions. It helps maintainers and others understand how to go about reviewing your contribution. For more tips, check out [Writing Your First Pull Request: Tips, Best Practices, and AI-Powered Tools for Success](https://dev.to/opensauced/writing-your-first-pull-request-tips-best-practices-and-ai-powered-tools-for-success-3bg9)

**Comment by Lymah123** [comment]
> Well done @Lymah123! :) I made some more edits to your PR's form. As you continue your open source journey, always fill out every section of the pull request form based on the commented instructions. It helps maintainers and others understand how to go about reviewing your contribution. For more tips, check out [Writing Your First Pull Request: Tips, Best Practices, and AI-Powered Tools for Success](https://dev.to/opensauced/writing-your-first-pull-request-tips-best-practices-and-ai-powered-tools-for-success-3bg9)

Noted!!! Thanks for always being patient with me @CBID2! I so much appreciate you.

**Pull request merged**
